cmd/link/link.go

```diff
@@ -26,12 +38,15 @@ If successful, the last line of the output will contain the link. Exact
 capabilities depend on the remote, but the link will always be created with
 the least constraints – e.g. no expiry, no password protection, accessible
 without account.
+
+Use the --unlink flag to remove existing public link to file/folder with link command rather than creating,
+Default is false, meaning link command will create or retrieve public link
```

Comment:
This needs a similar note: Not all backends support `--unlink` - those that don't will just ignore it.